src/test/java/org/jabref/logic/importer/util/GrobidServiceTest.java

```diff
@@ -0,0 +1,45 @@
+package org.jabref.logic.importer.util;
+
+import org.jabref.preferences.JabRefPreferences;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class GrobidServiceTest {
```

Comment:
Please tag this as a `FetcherTest` (because it relies on the network connection)

Reply:
1fd21914d0251cde13a7e3c001938ab1af2ad4bd